src/webhook/dto/create-webhook-request.ts

```diff
@@ -23,7 +23,9 @@ export class CreateWebhookRequest {
   @Transform(({ value }) => Array.from(new Set(value)))
   @IsIn(['recipe_created', 'recipe_updated', 'recipe_deleted'], { each: true })
   @ApiProperty({
-    enum: ['recipe_created', 'recipe_updated', 'recipe_deleted'],
+    type: String,
+    isArray: true,
+    example: ['recipe_created', 'recipe_updated', 'recipe_deleted'],
```

Comment:
how about using something like this (warning, might not be a valid code):

```typescript
type: 'array',
     items: {
            type: 'string'
         }
```
?

Reply:
Yep, it works as expected